test/functional/p2p_leak.py

```diff
@@ -70,7 +70,7 @@ class CNodeNoVersionBan(CLazyNode):
     # NOTE: implementation-specific check here. Remove if bitcoind ban behavior changes
     def on_open(self):
         super().on_open()
-        for i in range(banscore):
+        for _ in range(DISCOURAGEMENT_THRESHOLD):
```

Comment:
Why the change from 10 to 100 iterations here? :)

Reply:
Because we can no longer use `-banscore` to change the default threshold (see removed line in test: `self.extra_args = [['-banscore=' + str(banscore)]]`), so we have to use `DISCOURAGEMENT_THRESHOLD` here for the number of veracks sans version to send.

Note to self: the words "ban" in this functional test should be updated to "discourage".